test/controllers/tasks/active.psc.details.controller.unit.ts

```diff
@@ -1,3 +1,5 @@
+import {Templates} from "../../../src/types/template.paths";
```

Comment:
you don't have the githook running as it would have corrected the spacing around {Templates} automatically
it should be { Templates }

you need to run this on the command line
find .githooks -type f -exec ln -sf ../../{} .git/hooks/ \;

then try committing with {Templates} and you'll notice that it will change it automatically to { Templates }